core-libs/setup/ssr/logger/services/express-logger.service.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, inject } from '@angular/core';
+import { REQUEST } from '@nguniversal/express-engine/tokens';
+import { LoggerService } from '@spartacus/core';
+import { formatWithOptions } from 'util';
+import { serverLoggerToken } from '../loggers/server-logger';
+
+@Injectable({ providedIn: 'root' })
+export class ExpressLoggerService implements LoggerService {
+  request = inject(REQUEST);
+  logger = inject(serverLoggerToken);
```

Comment:
[MINOR]

To avoid ambiguity and for consistency, I suggest to rename it to `serverLogger = ...`